dashboard/common/utils.py

```diff
@@ -10,3 +18,18 @@ def get_build_info():
     build_name=os.getenv("OPENSHIFT_BUILD_NAME", "")
     return f'Build_Namespace:{build_namespace} Build_Name: {build_name} Git_Source: {build_source} Git_Branch: {git_branch} Git_Commit: {git_commit}'
 
+
+def get_resource_types():
+    resource_types = ENV.get("RESOURCE_TYPES")
+    return resource_types
+
+def get_resource_urls():
+    resource_urls = ENV.get("RESOURCE_URLS")
+    return resource_urls
+
+def get_resource_values():
+    resource_types = get_resource_types()
+    resource_values = []
+    for dict_value in resource_types.values():
```

Comment:
You don't need this loop or any of these temporary values.
You can should be able to just return from the first line

return get_resource_types().values()

Because of this is actually used in settings, I'd just move all this code to settings, you don't really need these methods here, just make them variables in settings

Cant just just remove the code here and change move it into settings as:
```
RESOURCE_TYPES = ENV.get("RESOURCE_TYPES")
RESOURCE_URLS = ENV.get("RESOURCE_URLS")
RESOURCE_VALUES = RESOURCE_TYPES.values() 
```